java-analyzer-bundle.test/projects/customers-tomcat-legacy/Makefile

```
@@ -0,0 +1,42 @@
+# These can be overidden with environment variables of the same name
+REGISTRY ?= quay.io
+REPOSITORY ?= rofrano
+IMAGE ?= customers-tomcat
+TAG ?= 0.1
+LOCAL_IMAGE_NAME ?= $(IMAGE):$(TAG)
+REMOTE_IMAGE_NAME ?= $(REGISTRY)/$(REPOSITORY)/$(IMAGE):$(TAG)
+
+all: build
```

checkmake (0.2.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)